storage_service/locations/models/package.py

```diff
@@ -619,8 +618,10 @@ def replicate(self, replicator_location_uuid):
                 replica_package.uuid,
                 master_checksum_algorithm,
             )
-            replication_validation_event = replica_package.get_replication_validation_event(
-                checksum_report=checksum_report, master_aip_uuid=self.uuid
+            replication_validation_event = premis.get_replication_validation_event(
+                replica_package,
```

Comment:
For consistency here, I think it would be good to see the form `replica_package_uuid=replica_package.uuid,` as with the other args. Is there a good reason to pass the entire object into the function at the moment, instead of just using the uuid?

Reply:
Very good point! I've changed to pass just the UUID (or other bits of data) for all PREMIS functions.